jerry-core/parser/js/js-parser.c

```diff
@@ -1712,6 +1715,7 @@ parser_post_processing (parser_context_t *context_p) /**< context */
   }
 
 #ifdef JERRY_ENABLE_LINE_INFO
+  JERRY_INIT_CURRENT_INSTANCE ();
```

Comment:
The macro is already used in a block above, there should be only one call to the `JERRY_INIT_CURRENT_INSTANCE` macro per method. So this and the previous macro call should be move before the start of the method.